lib/stages/FRBBeamformer.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef FRBBEAMFORMER_H
+#define FRBBEAMFORMER_H
+
+#include <string>
+
+#include "bufferContainer.hpp"
+#include "Config.hpp"
+#include "Stage.hpp"
+
+class FRBBeamformer : public kotekan::Stage {
```

Comment:
Haven't looked over this in detail yet, but you will want to use a `commandObject` (`cudaCommand` in this case) not a stage for invoking the kernel.  See for example: `lib/cuda/cudaCorrelatorAstron.cpp` or `lib/cuda/cudaShuffleAstron.cu`